Sources/NIOPerformanceTester/UDPBenchmark.swift

```diff
@@ -87,7 +90,7 @@ extension UDPBenchmark: Benchmark {
 }
 
 extension UDPBenchmark {
-    final class EchoHandler: ChannelInboundHandler {
+    final class EchoHandler: ChannelInboundHandler, Sendable {
```

Comment:
Do we want this to be `Sendable` if we have a `SendableView`?

Reply:
The `SendableView` is on a different type.